docs/usage/webhook/webhook.rst

```diff
@@ -0,0 +1,158 @@
+Redeploying on receving github webhook
+======================================
+
+We'll move  ``ffmpeg`` to a github repo,and set up the webhook dataflow such that whenever we push
+to the default branch,the new version is pulled and a docker container is build and run with this
+new version.
+
+Deploying on docker container
+-----------------------------
+
+We'll first setup a docker container for running ffmpeg http service.A ``Dockerfile`` is
+already generated in ffmpeg.We need to modify it to include ``ffmpeg``.
+
+**Dockerfile**
+
+.. literalinclude:: /../examples/ffmpeg/Dockerfile
+
+.. note::
+    The run command in the comment section of the Dockerfile will be used to execute
+    the container after receving webhooks,so make sure you change it to your usecase.
+
+For this tutorial we will change it to
+
+.. code-block:: console
+
+    docker run --rm -d -v ~/Videos:/usr/src/app/data -ti -p 8080:8080 aghinsa/ffmpeg \
+        -mc-config deploy -insecure -log debug
+
+you should replace ``aghinsa/ffmpeg`` with your ``USERNAME/REPONAME``,in both run and build commands,in cli
```

Comment:
The create script will have already replaced `REPLACE_ORG_NAME/REPLACE_IMPORT_PACKAGE_NAME` with their correct username.